doc/wave_calib.rst

```diff
@@ -33,6 +33,28 @@ XeI     4000-12000  14 April 2016
 ZnI     2900-8000   2 May 2016
 ======  ==========  =============
 
+By-Hand Calibration
+===================
+
+If the automatic algorithm is faililng (heaven forbid), you
```

Comment:
typo -- "failing"

Also, should we add a note that the user should contact the authors (so that we can investigate why the auto id algorithm did not work)?